contracts/Aavegotchi/libraries/LibAppStorage.sol

```diff
@@ -221,14 +221,14 @@ struct ERC721BuyOrder {
     bool[] validationOptions;
 }
 
-struct GotchiEquippedRecordsInfo {
-    uint256[EQUIPPED_WEARABLE_SLOTS] equippedRecordIds;
-    uint256 equippedRecordIdsCount;
+struct GotchiEquippedCommitmentsInfo {
+    uint256[EQUIPPED_WEARABLE_SLOTS] equippedCommitmentIds;
+    uint256 equippedCommitmentIdsCount;
```

Comment:
```suggestion
    uint256 equippedWearablesCount;
```

Reply:
This counter does not count the number of wearable equipped, instead it counts the number of commitment equipped.